### ISSUE TITLE
Annotations missing in devel

### ISSUE THREAD

**Issue by cpetig** [cpetig/tflite_micro_compiler#5]
in devel the comments are gone (which inputs and outputs are used for each invoke call), also it is no longer visible which operation is called (fix by defining and using an enum into g_registration)

**Comment by rafzi** [comment]
Wouldn't these changes conflict with the vision of having loops instead of linear code?

**Comment by cpetig** [comment]
Good point. I will probably need to put them into a different place (e.g. annotations on the arrays instead of on the code (which will probably be gone soon))

**Comment by cpetig** [comment]
Related to #13

**Comment by cpetig** [comment]
After implementing the reworked code I can only envision a comment with pseudo-code describing the network in human readable form:
E.g.
/*    Inputs  10 (1,1)
   0:  1 = QUANTIZE(10)
   1:  4 = FULLY_CONNECTED(1,2,3)
   2:  7 = FULLY_CONNECTED(4,5,6)
   3:  0 = FULLY_CONNECTED(7,8,9)
   4:  11 = DEQUANTIZE(0)
  Outputs  11 (1.1)  */
Do you think that this adds value to the generated file?

**Comment by rafzi** [comment]
Since the comment would be generated, I don't see any downside. It can only help with debugging!
